drift/test/manager/manager_prefetch_test.dart

```diff
@@ -0,0 +1,331 @@
+// ignore_for_file: unused_local_variable
+
+import 'package:drift/drift.dart' hide isNull, isNotNull;
+import 'package:test/test.dart';
+
+import '../generated/todos.dart';
+import '../test_utils/test_utils.dart';
+
+void main() {
+  late TodoDb db;
+  late List<StoreData> stores;
+  late List<DepartmentData> departments;
+  late List<ProductData> products;
+  late List<int> listings;
+  final categoryData = [
+    (description: "School", priority: Value(CategoryPriority.high)),
+    (description: "Work", priority: Value(CategoryPriority.low)),
+  ];
+  late int schoolCategoryId;
+  late int workCategoryId;
+  late List<
+      ({
+        Value<RowId> category,
+        String content,
+        Value<TodoStatus> status,
+        Value<DateTime> targetDate,
+        Value<String> title
+      })> todoData;
+
+  setUp(() async {
+    db = TodoDb(testInMemoryDatabase());
+    stores = [
```

Comment:
Can we share this data between tests to avoid all this lengthy setup stuff here? Having too many rows can also make the tests harder to understand because it makes it non-trivial to figure out what the right response should be.